packages/fxa-auth-db-mysql/db-server/test/backend/db_tests.js

```diff
@@ -3539,6 +3575,7 @@ module.exports = function(config, DB) {
               'correct emailCode'
             );
             assert.isTrue(!!res[1].isVerified);
+            assert.isAbove(res[1].verifiedAt, res[1].createdAt);
```

Comment:
@vbudhram how're we getting two emails [here](https://github.com/mozilla/fxa/blob/43cb00c79dfac383fdb3958e5b8b05576042c0c7/packages/fxa-auth-db-mysql/db-server/test/backend/db_tests.js#L3528) in the response when we're just creating only 1 email, the first email is having the `isVerified` set as `true` but the `verifiedAt` field is still `null`. I tried to find out why we've 2 emails in the response and only that can answer this that why `verifiedAt` is still null, may be we've to modify some other db procedure also that is setting the `isVerified` true.